<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>482</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCommandLinkButton" name="fusionButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>350</y>
      <width>185</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Execute Fusion</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="openFilesButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>350</y>
      <width>185</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Open Image Files</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="optionButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>390</y>
      <width>185</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Options</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>390</y>
      <width>185</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>531</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="HL1"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>210</y>
      <width>531</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="HL2"/>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>quitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>601</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
